CMakeLists.txt

```diff
@@ -185,11 +185,23 @@ endif()
 if (CMAKE_CONFIGURATION_TYPES)
   foreach (build_type IN LISTS CMAKE_CONFIGURATION_TYPES)
     execute_process(COMMAND "${CMAKE_COMMAND}" -E make_directory "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include")
-    execute_process(COMMAND "${CMAKE_COMMAND}" -E ${LINK_COMMAND} "${CMAKE_CURRENT_SOURCE_DIR}/runtime" "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include/runtime")
+    if (CMAKE_HOST_WIN32)
+      file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/runtime" _srcDir)
+      file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include/runtime" _dstDir)
+      execute_process(COMMAND cmd.exe /c mklink /J "${_dstDir}" "${_srcDir}")
+    else()
+      execute_process(COMMAND "${CMAKE_COMMAND}" -E ${LINK_COMMAND} "${CMAKE_CURRENT_SOURCE_DIR}/runtime" "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include/runtime")
+    endif()
   endforeach()
 else()
   execute_process(COMMAND "${CMAKE_COMMAND}" -E make_directory "${CMAKE_CURRENT_BINARY_DIR}/include")
-  execute_process(COMMAND "${CMAKE_COMMAND}" -E ${LINK_COMMAND}  "${CMAKE_CURRENT_SOURCE_DIR}/runtime" "${CMAKE_CURRENT_BINARY_DIR}/include/runtime")
+  if (CMAKE_HOST_WIN32)
+    file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/runtime" _srcDir)
+    file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}/include/runtime" _dstDir)
+    execute_process(COMMAND cmd.exe /c mklink /J "${_dstDir}" "${_srcDir}")
```

Comment:
What about the cmake/install-rules.cmake parts that use the link command?

Do those work on your machine? 

```
cmake -B build -T ClangCL -DCMAKE_INSTALL_PREFIX=jakt-install
cmake --build build
cmake --install build
```

Or do we need to do the same check there?

Reply:
That part doesn't complain for some reason, I don't really understand why it would be any different.